src/main/webapp/script.js

```diff
@@ -736,6 +736,3 @@ function transformAgregatedDataforGeoChart(aggregatedData) {
     google.charts.setOnLoadCallback(drawRegionsMap(array, aggregatedData)); 
   });
 }
```

Comment:
```suggestion
}
    module.exports = {getTotalCosts, getAverageCosts, getFailedJobs, getFailedJobsCost, getAveragevCPUCount, SSDVsHDDTimeComparison};
```  

Not entirely sure that the indentation was right as it's a little awkward to add but here you go :D